### PR TITLE
Add time to waypoints and fix bug

### PR DESCRIPTION
**Pull request by merlos**
Hi!

I was trying to use this gem in a project and I found some issues:
- Waypoints did not include time element
- There was a problem in GPXFile.write
- While trying to upload the output of the GPX file to open street maps (web editor), it required to add xmlns to xsi attribute.

Also, I added some doc about how to create a gpx file on the readme. In my case I was not interested in parsing the file but in exporting some active record data.

I read you tried to contact the original author, did you get any response about the license?

Thanks
Best, Juan.

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -29,6 +29,49 @@ Converting a Magellan track log to GPX:
          GPX::MagellanTrackLog::convert_to_gpx(filename, "#{filename}.gpx")
       end
 
+Exporting an ActiveRecord to GPXFile (as Waypoints)
+
+    #
```
Comment (andrewhao) [inline README.rdoc]: Would you mind wrapping this code sample in a [Ruby GFM code fence](https://help.github.com/articles/github-flavored-markdown/#syntax-highlighting)? That way we can have syntax highlighting on the following lines.

**Comment by andrewhao** [comment]
Please see comment around minor nits around README formatting. Otherwise, looks good. Thanks for the tests. I'll merge when you get a chance to address the README.

**Comment by andrewhao** [comment]
@merlos I have not heard back from @dougfales, but I'm going to try to contact him again manually.

**Comment by merlos** [comment]
Thanks Andrew for the reply.

I've done the changes. Check them out. The repo I used as base still had readme.rdoc.  I've renamed it to .md (I saw you had a branch with that change).

Regards, Juan.

**Comment by andrewhao** [comment]
Sounds good. When these builds pass, I'll merge and release a new version of the gem for you.

**Comment by andrewhao** [comment]
All right, I released 0.8. Have a look, @merlos.

**Pull request merged**